model/Export/AbstractQTIItemExporter.php

```diff
@@ -316,4 +326,26 @@ protected function getMetadataExporter()
         }
         return $this->metadataExporter;
     }
+
+    private function copySharedStimulusCssFiles(string $link, string $basePath): string
+    {
+        $asset = new core_kernel_classes_Resource(UriHelper::decode($link));
+        $baseName = 'assets/' . $this->getSharedStimulusCSSExporter()->getResourceLink($asset);
```

Comment:
let's save `assets` into a constant and put directory_Separator or separate the '/'

Reply:
Related to this: https://github.com/oat-sa/extension-tao-mediamanager/pull/276/files#diff-dfe8daf8abc904a842e60b83b67c3aac0ccaa187cb4fc205c5bafc6fa07fab3cR162
Why you removed the $link param from `pack` method then added `getResourceLink` to `SharedStimulusCSSExporter` and again use it from here to specify zipPath but do not using initial resourceLink from $assets 3 lines above? Is it a different resource? I suggest to keep separate a new logic related to the Item export and adapt income data outside  `SharedStimulusCSSExporter` to keep it's responsibility clear and avoid breaking changes